cdap-runtime-ext-emr/src/main/resources/aws-emr.json

```diff
@@ -0,0 +1,197 @@
+{
+  "configuration-groups": [
+    {
+      "label": "AWS Account Information",
+      "properties": [
+        {
+          "widget-type": "securekey-textarea",
+          "label": "Access Key",
+          "name": "accessKey",
+          "required": true,
+          "description": "An AWS Access Key ID identifies an AWS access key, which can be used to make secure REST or HTTP query protocol requests to AWS service APIs.",
+          "widget-attributes": {
+            "placeholder": "AWS Access Key"
+          }
+        },
+        {
+          "widget-type": "securekey-textarea",
+          "label": "Secret Key",
+          "name": "secretKey",
+          "required": true,
+          "description": "An AWS Secret Key can be used to make secure REST or HTTP query protocol requests to AWS service APIs.",
+          "widget-attributes": {
+            "placeholder": "AWS Secret Key"
+          }
+        }
+      ]
+    },
+    {
+      "label": "General Settings",
+      "properties": [
+        {
+          "widget-type": "select",
+          "label": "Region",
+          "name": "region",
+          "description": "When you launch an Amazon EMR cluster, you must specify a region. You might choose a region to reduce latency, minimize costs, or address regulatory requirements. For more information, refer to https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-region.html.",
+          "widget-attributes": {
+            "values_sourced_from": "https://docs.aws.amazon.com/general/latest/gr/rande.html#emr_region",
```

Comment:
didn't know this was a thing. Or are you just documenting where it came from?

Reply:
I was just documenting where it came from. JSON format doesn't allow comments.